### ISSUE TITLE
Bybit fetch_balance for tradingBot account

### ISSUE THREAD

**Issue by Gorgias26** [ccxt/ccxt#19092]
### Operating System

Windows

### Programming Languages

Python

### CCXT Version

4.0.80

### Description

Hello when I use the command: fetch_balance 

I can't find the funds that are currently used in the bybit trading bot tools.

I use  `exchange.options['defaultType'] = 'funding' `
to find the funds in the funding account, I have tried many things to find the funds used in the trading bot. 
It's not in SPOT/SWAP/FUTURE

So I'm asking for your help if you've found the solution?

Thank you in advance

### Code

```
  

```

**Comment by carlosmiei** [comment]
Hello @Gorgias26,
If you're trying to access the funding account, you should do this instead `await fetch_balance({'type':'funding'})`

**Comment by Gorgias26** [comment]
@carlosmiei  thank you for your help. Buti'm sorry, i wasn't clear. i'm talking about the bybit trading bot tool.

like this : [https://gyazo.com/159e736cf53d4dff4cc6aad6fa35d03f](url)
![159e736cf53d4dff4cc6aad6fa35d03f](https://github.com/ccxt/ccxt/assets/37674056/cb933143-8267-4eb8-a61e-0c3b3af45139)



I gave info on the funding portfolio, because that's where the funds go after a TP or SL. That's why I put what I found in my post if it can help others, who are looking to automate bybit trading bot.

But I couldn't find the solution to access the funds invested in the trading bot proposed by bybit using the api.
